test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
Added Mvc for AntiForgery. Instead of adding entire MVC, we can also add a set of services needed as follows:

            services.AddDataProtection();
            services.AddSingleton<IClaimUidExtractor, DefaultClaimUidExtractor>();
            services.AddSingleton<AntiForgery, AntiForgery>();
            services.AddSingleton<IAntiForgeryAdditionalDataProvider, DefaultAntiForgeryAdditionalDataProvider>();

@harshgMSFT and @Praburaj,
Which one do you think we should recommend to customers?